lib/qxcli/commands/contrib/Update.js

```diff
@@ -186,7 +186,7 @@ qx.Class.define("qxcli.commands.contrib.Update", {
               }
               // we have a list of Manifest.json paths!
               if (data.contribs && data.contribs instanceof Array) {
-                manifests = data.contribs;
+                manifests = data.libraries || data.contribs; // data.contribs is deprecated
```

Comment:
if it's deprecated, can we just not have it?

Reply:
It used to be called "contribs", but I think "libraries" is much more general - there is only one contrib that contains a `qooxdoo.org` file with `contribs` (the qookery project) and I did not want to break that. Once they have updated their code, we can remove `contribs`